### ISSUE TITLE
Problemas cocoon + slides apresentação

### ISSUE THREAD

**Issue by renatopanda** [nelsonmpg/IPTomar-MEI-IoT-ES#44]
A presentation tem uma relação "has_one" com project.

Usar cocoon neste caso gera alguns problemas: já limitei (+-) inserção de slides a 1 ficheiro e corrigi o erro que faz com que ao editar ele apague a apresentação existente.

Continua a não funcionar correctamente. Quando se apaga o pdf dos slides também vai ser removida a informação da sala e data (o que não é suposto). Neste caso não faz sentido usar cocoon, só existe um ficheiro associado, não precisamos de nested fields dinâmicos.

**Comment by MiguelFCoelho** [comment]
Vi que o professor fez alterações no commit a seguir, mas o problema continua por resolver?

**Comment by renatopanda** [comment]
Sim, a única coisa que é feita é bloquear o botão "adicionar slides" quando existe um ficheiro. A questão é que, podemos apagar slides e querer manter a sala e data, mas neste caso ao guardar o formulário e eliminar os slides ele elimina o resto também.
